packages/eslint-plugin-emotion/src/rules/pkg-renaming.js

```diff
@@ -0,0 +1,32 @@
+let simpleMappings = {
```

Comment:
I know it's a draft at this point but we should also handle that more complex stuff - moved theming, removed `@emotion/css`, and also macro versions